nifi-extension-bundles/nifi-groovyx-bundle/nifi-groovyx-processors/pom.xml

```diff
@@ -71,6 +67,12 @@
             <artifactId>nifi-record</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-dbcp-service</artifactId>
```

Comment:
Is this dependency necessary for testing? The test extends the Lookup class, but it seems like it should be sufficient to mock the `DBCPService` interface.